build.rs

```diff
@@ -34,6 +34,8 @@ fn build_engine(src: &PathBuf, out_dir: &Path) {
     let makefile_path = src.join("js-compute-runtime").join("Makefile");
     let status = Command::new("make")
         .current_dir(out_dir)
+        .arg("-j")
+        .arg("2")
```

Comment:
I'd like to pass `-j ${num_cpus}` here, and can do that using the `num_cpus` crate. Cargo already tracks this via the `cargo.jobs` config variable, but it's not clear to me how to get the value of that variable from the context of a build script.

Reply:
The `NUM_JOBS` env var is what I was looking for here.

https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts